doc/common/spellcheck.md

```diff
@@ -1,6 +1,10 @@
-
 # Spell Checking the Azure SDK
 
+> [!NOTE]  
+> If you are working on an Azure REST API spec see SpellCheck instructions here: 
+> [https://aka.ms/ci-fix#spell-check](https://aka.ms/ci-fix#spell-check)
```

Comment:
I'm fine with this change but I think we should probably move those docs into here at some point as I want a similar pattern to be used in other repos.